cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -1317,11 +1317,43 @@ declare namespace Deno {
    */
   export function openKv(path?: string): Promise<Deno.Kv>;
 
+  export interface Schedule {
+    minute?: number | { exact: number | number[] } | {
+      start?: number;
+      end?: number;
+      every?: number;
+    };
```

Comment:
Since it's the same pattern repeated for every field, it makes sense to extract this into a type. something like this:
```ts
type CronScheduleExpression = number | { exact: number | number[] } | {
  start?: number;
  end?: number;
  every?: number;
};
```

Reply:
Done.